model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -503,11 +558,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "INFO:tensorflow:Model is compatible with the `Dataset` instance.\n",
+      "WARNING:tensorflow:Using past predictions. the data must match exactly (same number of lines and same order).\n"
+     ]
+    }
+   ],
    "source": [
-    "dataset_performance.add_model_prediction_to_data(model)"
+    "dataset_performance.add_model_prediction_to_data(model, recompute_predictions=False)"
```

Comment:
I am wondering if we want to leave this parameter as False or True. False avoids extra computation (in particular for someone that does not know the parameter) but it would also lead to an error when using a new model.

Please do what you think is best!